src/main/java/frc/robot/subsystems/climber/ClimberIOTalonFX.java

```diff
@@ -0,0 +1,304 @@
+    inputs.rightSupplyCurrentAmps = rightSupplyCurrentAmpsStatusSignal.getValueAsDouble();
+    inputs.rightCurrentSetpoint = rightSetpointStatusSignal.getValueAsDouble();
+
+    inputs.leftReferenceSetpointMeters = leftRequestedReferenceSetpointMeters;
+    inputs.leftSupplyCurrentAmps = leftRequestedSupplyAmps;
+
+    inputs.rightReferenceSetpointMeters = rightRequestedReferenceSetpointMeters;
+    inputs.rightSupplyCurrentAmps = rightRequestedSupplyAmps;
+
+    // inputs.closedLoopError = leftMotor.getClosedLoopError().getValue();
+    // inputs.power =
+    // inputs.controlMode = leftMotor.getControlMode().toString();
+
+    // update configuration if tunables have changed
+    if (leftMotorKP.hasChanged()
+        || leftMotorKI.hasChanged()
+        || leftMotorKD.hasChanged()
+        || leftMotorKS.hasChanged()
+        || rightMotorKP.hasChanged()
+        || rightMotorKI.hasChanged()
+        || rightMotorKD.hasChanged()
+        || rightMotorKS.hasChanged()) {
+      TalonFXConfiguration config = new TalonFXConfiguration();
+      MotionMagicConfigs motionMagicConfigs = new MotionMagicConfigs();
+
+      this.leftMotor.getConfigurator().refresh(config);
+      config.Slot0.kP = leftMotorKP.get();
+      config.Slot0.kI = leftMotorKI.get();
+      config.Slot0.kD = leftMotorKD.get();
+      config.Slot0.kS = leftMotorKS.get();
+      config.Slot0.kV = leftMotorKV.get();
+      config.Slot0.kA = leftMotorKA.get();
+      motionMagicConfigs.MotionMagicExpo_kV = leftMotorKVExpo.get();
+      motionMagicConfigs.MotionMagicExpo_kA = leftMotorKAExpo.get();
+
+      this.leftMotor.getConfigurator().apply(config);
+
+      this.rightMotor.getConfigurator().refresh(config);
+      config.Slot0.kP = rightMotorKP.get();
+      config.Slot0.kI = rightMotorKI.get();
+      config.Slot0.kD = rightMotorKD.get();
+      config.Slot0.kS = rightMotorKS.get();
+      config.Slot0.kV = rightMotorKV.get();
+      config.Slot0.kA = rightMotorKA.get();
+      motionMagicConfigs.MotionMagicExpo_kV = rightMotorKVExpo.get();
+      motionMagicConfigs.MotionMagicExpo_kA = rightMotorKAExpo.get();
+      this.rightMotor.getConfigurator().apply(config);
+    }
+  }
+
+  /**
+   * Set the leftMotor position to the specified value in degrees.
+   *
+   * @param position the position to set the leftMotor to in degrees
+   * @param arbitraryFeedForward the arbitrary feed forward as a percentage of maximum power
+   */
+  @Override
+  public void setLeftMotorPosition(double position) {
+    leftMotor.setControl(
+        leftPositionCurrentRequest
+            .withPosition(Conversions.degreesToFalconRotations(position, GEAR_RATIO))
+            .withFeedForward(POSITION_PID_KG)); 
+  }
+
+  @Override
+  public void setRightMotorPosition(double position) {
+    this.rightMotor.setControl(
+        rightPositionCurrentRequest
+            .withPosition(Conversions.degreesToFalconRotations(position, GEAR_RATIO))
+            .withFeedForward(POSITION_PID_KG));
```

Comment:
Specify kG when setting other feedforward values instead of invoking this method.